dns-record-check/README.md

```diff
@@ -0,0 +1,35 @@
+# Chainlink External Adapter to Check DNS Records
+
+DNS Adapter lets query DNS over HTTPS (DoH) and check whether some record provided exists
+
+## Configuration
+
+The adapter takes the following environment variables:
+
+- `DNS_PROVIDER`: DNS provider to use. Options available:
+    - `cloudfare`
+    - `google`
+
+## Input Params
+
+- `name`: Query Name, eg. "example.com"
+- `type`: Query Type (either a numeric value or text), eg. "TXT"
+- `do` (optional): DO bit - set if client wants DNSSEC data (either boolean or numeric value), eg. "true"
+- `cd` (optional): CD bit - set to disable validation (either boolean or numeric value), eg. "false"
```

Comment:
This `dns-record-check` adapter uses the underlying `dns-query` adapter. We don't need to duplicate the documentation of `dns-query` here in `dns-record-check` package. We should just link to the underlying adapter README, and point to stuff that should be configured but documented there.